signer/tests/integration/transaction_coordinator.rs

```diff
@@ -1791,6 +1784,672 @@ async fn sign_bitcoin_transaction() {
     }
 }
 
+/// Test that three signers can successfully sign and broadcast a bitcoin
+/// transaction where the inputs are locked by different aggregate keys.
+///
+/// The test setup is as follows:
+/// 1. There are three "signers" contexts. Each context points to its own
+///    real postgres database, and they have their own private key. Each
+///    database is populated with the same data.
+/// 2. Each context is given to a block observer, a tx signer, and a tx
+///    coordinator, where these event loops are spawned as separate tasks.
+/// 3. The signers communicate with our in-memory network struct.
+/// 4. A real Emily server is running in the background.
+/// 5. A real bitcoin-core node is running in the background.
+/// 6. Stacks-core is mocked.
+///
+/// In this test the signers do quite a few things:
+/// 1. Run DKG
+/// 2. Sign and broadcast a rotate keys transaction.
+/// 3. Sweep some deposited funds into their UTXO.
+/// 4. Mint sBTC to the recipient.
+/// 5. Run DKG again.
+/// 6. Sweep two more deposits in one bitcoin transaction. These deposits
+///    are locked with different aggregate keys.
+/// 7. Have the signers UTXO locked by the aggregate key from the second
+///    DKG run.
+///
+/// For step 6, we "hide" the DKG shares to get the signers to run DKG
```

Comment:
Step 5?

Reply:
Oh I deleted a step and forgot to update.